marimo/_plugins/stateless/pdf.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2023 Marimo. All rights reserved.
+from __future__ import annotations
+
+import base64
+import io
+from typing import Any, Optional, Union
+
+from marimo._output.builder import h
+from marimo._output.hypertext import Html
+from marimo._output.rich_help import mddoc
+from marimo._output.utils import create_style
+
+
+@mddoc
+def pdf(
+    src: Union[str, io.IOBase],
+    initial_page: Optional[int] = None,
+    width: Optional[Union[int, str]] = "100%",
+    height: Optional[Union[int, str]] = "70vh",  # arbitrary, but good default
```

Comment:
`Optional` means can be `None`, not can be omitted Is that intended here? If not then should remove `Optional`

Reply:
yea I wanted the user to be allowed to unset the default if it's useful. I personally don't think they will want to or need to , but it could come from and outside value